system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -528,7 +528,7 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 					RuntimeCall::Uniques { .. } |
 					RuntimeCall::Scheduler(..) |
 					RuntimeCall::Treasury(..) |
-					RuntimeCall::Bounties(..) |
+					//RuntimeCall::Bounties(..) | # TODO more
```

Comment:
Q here as i asked in the chat on whether to change proxies or not. Am thinking about changing them now.